tools/clang/lib/SPIRV/EmitVisitor.cpp

```diff
@@ -431,7 +431,9 @@ void EmitVisitor::initInstruction(SpirvInstruction *inst) {
                                spv::Decoration::RelaxedPrecision, {});
   }
   // Emit NoContraction decoration (if any).
-  if (inst->isPrecise() && inst->isArithmeticInstruction()) {
+  // STEVEN: Check if Gis is set here.
```

Comment:
Is this TODO a leftover?

Reply:
Yes tha was a marker to remind where to add the code after I added the new option. No longer needed.